cmd/dex/config.go

```diff
@@ -119,6 +125,46 @@ type GRPC struct {
 	TLSClientCA string `json:"tlsClientCA"`
 }
 
+type Userinfo struct {
+	Type   string         `json:"type"`
+	Config UserinfoConfig `json:"config"`
```

Comment:
Better name it DRDConfig or why not just Config?

Reply:
see other comment. This is rerally meant to be more generic